core/src/iosMain/kotlin/com/revenuecat/purchases/kmp/Purchases.ios.kt

```diff
@@ -36,7 +59,17 @@ public actual class Purchases private constructor(private val iosPurchases: IosP
                 ?: IosPurchases.logHandler().toLogHandler().also { _logHandler = it }
             set(value) {
                 _logHandler = value
-                IosPurchases.setLogHandler(value.toRcLogHandler())
+                IosPurchases.setLogHandler { level, message ->
+                    val tag = "PurchasesKMP"
+                    when (level) {
+                        RCLogLevelVerbose -> value.v(tag, message ?: "")
+                        RCLogLevelInfo -> value.d(tag, message ?: "")
+                        RCLogLevelDebug -> value.d(tag, message ?: "")
+                        RCLogLevelWarn -> value.w(tag, message ?: "")
+                        RCLogLevelError -> value.e(tag, message ?: "", null)
+                        else -> value.e(tag, "Unrecognized log level", null)
+                    }
```

Comment:
Same comment as on the Android side. Maybe this can be non-anonymous in `LogHandler.ios.kt`, and maybe we can keep a reference to the original instance passed, so we can return that in `get`?

Reply:
Resolved with #162.